priv/graphql/schemas/global/vcard.gql

```diff
@@ -0,0 +1,271 @@
+    "Email address"
+    userId: String
+}
+
+type GeographicalPosition{
+    "Geographical latitude"
+    lat: String
+    "Geographical longtitude"
+    lon: String
+}
+
+type Organization{
+    "Organization name"
+    orgname: String
+    "Organization unit"
+    orgunit: [String]
+}
+
+type Privacy{
+    tags: [PrivacyClassificationTags]
+}
+
+type Key{
+    credential: String
+}
+
+input VcardInput{
+    "Formatted name from Vcard"
+    formattedName: String!
+    "Person's name details"
+    nameComponents: NameComponentsInput!
+    "User's nickname"
+    nickname: [String]
+    "User's photo"
+    photo: [String!]
+    "Birthday date"
+    birthday: [String!]
+    address: [AddressInput!]
+    label: [LabelInput!]
+    telephone: [TelephoneInput!]
+    email: [EmailInput!]
+    jabberId: [String!]
+    mailer: [String!]
+    timeZone: [String!]
+    "Geographical position"
+    geo: [GeographicalPositionInput!]
+    title: [String!]
+    role: [String!]
+    logo: [String!]
+    agent: [String!]
+    "Organization"
+    org: [OrganizationInput!]
+    categories: [KeywordInput!]
+    note: [String!]
+    "Identifier of product that generated the vCard property"
+    prodId: [String!]
+    "Last revised property. The value must be an ISO 8601 formatted UTC date/time"
+    rev: [String!]
+    "Sort string property"
+    sortString: [String!]
+    "Formatted name pronunciation property"
+    sound: [String!]
+    "Unique identifier property"
+    uid: [String!]
+    "Directory URL property"
+    url: [String!]
+    "Free-form descriptive text"
+    desc: [String!]
+    "Privacy classification property"
+    class: [PrivacyInput!]
+    "Authentication credential or encryption key property"
+    key: [KeyInput!]
+}
+
+input KeywordInput{
+    keyword: [String!]
+}
+
+input NameComponentsInput{
+    family: String
+    givenName: String
+    middleName: String
+    prefix: String
+    suffix: String
+}
+
+input AddressInput{
+    "Address tags"
+    tags: [AddressTags!]
+    "Post office box"
+    pobox: String
+    "Extra address data"
+    extadd: String
+    street: String
+    locality: String
+    region: String
+    "Postal code"
+    pcode: String
+    country: String
+}
+
+input LabelInput{
+    "Label tags"
+    tags: [AddressTags!]
+    line: [String!]!
+}
+
+input TelephoneInput{
+    "Telephone tags"
+    tags: [TelephoneTags!]
+    number: String!
+}
+
+input EmailInput{
+    "Email tags"
+    tags: [EmailTags!]
+    "Email address"
+    userId: String!
+}
+
+input GeographicalPositionInput{
+    "Geographical latitude"
+    lat: String!
+    "Geographical longtitude"
+    lon: String!
+}
+
+input OrganizationInput{
+    "Organization name"
+    orgname: String!
+    "Organization unit"
+    orgunit: [String!]
+}
+
+input PrivacyInput{
+    tags: [PrivacyClassificationTags!]
+}
+
+input KeyInput{
+    credential: String!
+}
+
+enum PrivacyClassificationTags{
+    PUBLIC
+    PRIVATE
+    CONFIDENTIAL
+}
+
+enum AddressTags{
```

Comment:
NOTE:
In documentation is stated:
```xml
 (DOM | INTL)?,
```
Thus we cannot have these two tags together. I think it is a minor thing, but it's good to know that the user is allowed to add them together through our API.

Second thing is that GraphQL doesn't have the type for a unique list. So it is possible to put a list with a duplication even when vcard documentation requires only one.